<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1295</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1291</width>
      <height>751</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>ASCII Art Generator</string>
     </attribute>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>1261</width>
        <height>571</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Console</family>
        <pointsize>8</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>20</y>
        <width>453</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="selectSizeLabel">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Select size of ASCII art:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="widthLineEdit">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>100</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="maxSizeLabel">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Max size: 1000</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="uploadImageLabel">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Upload an image:</string>
           </property>
           <property name="margin">
            <number>9</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonASCII">
           <property name="text">
            <string>Open File</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="fileTypesLabel">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>File types supported: .png, .jpg, .bmp</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="errorLabelASCII">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>10</y>
        <width>351</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Filters</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>50</y>
        <width>451</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="uploadImageLabel_2">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Upload an image:</string>
         </property>
         <property name="margin">
          <number>9</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonFilters">
         <property name="text">
          <string>Open File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="fileTypesLabel_2">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>File types supported: .png, .jpg, .bmp</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="errorLabelFilters">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>20</y>
        <width>301</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="imageLabelFilters">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>110</y>
        <width>721</width>
        <height>431</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>910</x>
        <y>140</y>
        <width>256</width>
        <height>56</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="radioButtons">
         <item>
          <widget class="QRadioButton" name="noFilterRadioButton">
           <property name="text">
            <string>No filter</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="blackAndWhiteRadioButton">
           <property name="text">
            <string>Black and White</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="blurRadioButton">
           <property name="text">
            <string>Blur</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="radioButtons_2">
         <item>
          <widget class="QRadioButton" name="contrastRadioButton">
           <property name="text">
            <string>Contrast</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="saturationRadioButton">
           <property name="text">
            <string>Saturation</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="temperatureRadioButton">
           <property name="text">
            <string>Temperature</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QSlider" name="blackAndWhiteSlider">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>330</y>
        <width>281</width>
        <height>16</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="minimum">
       <number>-1000</number>
      </property>
      <property name="maximum">
       <number>1000</number>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="blackAndWhiteMin">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>340</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>  0</string>
      </property>
     </widget>
     <widget class="QLabel" name="blackAndWhiteMax">
      <property name="geometry">
       <rect>
        <x>1220</x>
        <y>340</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>  1</string>
      </property>
     </widget>
     <widget class="QSlider" name="nonNormalSlider">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>390</y>
        <width>281</width>
        <height>16</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="maximum">
       <number>5000</number>
      </property>
      <property name="value">
       <number>1000</number>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="nonNormalMin">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>410</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="nonNormalMax">
      <property name="geometry">
       <rect>
        <x>1220</x>
        <y>410</y>
        <width>16</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>  5</string>
      </property>
     </widget>
     <widget class="QLabel" name="nonNormalMax_2">
      <property name="geometry">
       <rect>
        <x>1000</x>
        <y>410</y>
        <width>16</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>  1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="downloadPushButton">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>520</y>
        <width>111</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Download Image</string>
      </property>
     </widget>
     <widget class="QLabel" name="blackAndWhiteMax_2">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>340</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>  -1</string>
      </property>
     </widget>
     <widget class="QLabel" name="noticeLabelFilters">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>220</y>
        <width>301</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1295</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
